### PR TITLE
Add TransferEndpointType enum example to docs

### PR DESCRIPTION
**Pull request by sirosen**

### CODE REVIEW

File: docs/examples/endpoint_type_enum.rst [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            else:
+                gcsv5 = False
+
+            if gcsv5:
+                if has_host:
+                    return EndpointType.GUEST_COLLECTION
+                else:
+                    return EndpointType.MAPPED_COLLECTION
+
+            elif shared:
```
Comment (aaschaer) [inline docs/examples/endpoint_type_enum.rst:84]: I think this should be `has_host`

**Review by aaschaer — COMMENTED** [review]
Found a few issues in the example code.

There seems to be mixed usage between EndpointType and TransferEndpointType. Not sure which one is "correct" but only one should be used

File: docs/examples/endpoint_type_enum.rst (hunk 1 above)
Comment (sirosen) [reply docs/examples/endpoint_type_enum.rst:84]: Yes, thank you for the save. I was editing the CLI version of this (where we called it `shared`) and trying to adjust the names for a broader audience.

**Comment by sirosen** [comment]
@aaschaer, could you take a second look at this?

I've just corrected the issues you pointed out, and I think this is now clean.

**Review by aaschaer — APPROVED** [review]

**Pull request merged**